### PR TITLE
[MOB-10131] remove duplicate code

### PR DESCRIPTION
**Pull request by lposen**
## 🔹 JIRA Ticket(s) if any

* [MOB-10131](https://iterable.atlassian.net/browse/MOB-10131)

## ✏️ Description

Removes duplicate code from repository

[MOB-10131]: https://iterable.atlassian.net/browse/MOB-10131?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/IterableAction.ts [hunk 1]
```diff
@@ -1,3 +1,25 @@
...
+ */
+export enum IterableLogLevel {
+  debug = 1,
+  info = 2,
+  error = 3,
+}
+
+/**
+ * IterableAction represents an action defined as a response to user events.
+ * It is currently used in push notification actions (open push & action buttons).
```
Comment (evantk91) [inline src/IterableAction.ts:21]: This is also used for action buttons for in-app and embedded.

Comment (evantk91) [reply src/IterableAction.ts:21]: We should also add a description for `IterableActionContext`

**Review by evantk91 — APPROVED** [review]
Added some nit picks.

File: src/IterableAction.ts (hunk 1 above)
Comment (lposen) [reply src/IterableAction.ts:21]: It's done in a future PR

**Pull request merged**